src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -312,36 +234,31 @@ BaselinesTable.propTypes = {
     baselineTableData: PropTypes.array,
     createBaselinesTable: PropTypes.func,
     selectBaseline: PropTypes.func,
-    selectOneBaseline: PropTypes.func,
     addSystemModalOpened: PropTypes.bool,
     fetchBaselines: PropTypes.func,
-    history: PropTypes.object,
+    tableType: PropTypes.string,
+    onClick: PropTypes.func,
     kebab: PropTypes.bool,
     createButton: PropTypes.bool,
     exportButton: PropTypes.bool,
     createBaselineModalOpened: PropTypes.bool,
-    hasSelect: PropTypes.bool,
-    hasSingleSelect: PropTypes.bool,
-    clearSelectedBaselines: PropTypes.func
+    onSelect: PropTypes.func,
+    clearSelectedBaselines: PropTypes.func,
+    columns: PropTypes.array
 };
 
 function mapStateToProps(state) {
     return {
         baselineDeleteLoading: state.baselinesTableState.baselineDeleteLoading,
-        baselineListLoading: state.baselinesTableState.baselineListLoading,
-        baselineTableData: state.baselinesTableState.baselineTableData,
-        addSystemModalOpened: state.addSystemModalState.addSystemModalOpened,
-        createBaselineModalOpened: state.createBaselineModalState.createBaselineModalOpened
+        addSystemModalOpened: state.addSystemModalState.addSystemModalOpened
```

Comment:
Looks like you can drop this and drop all of `mapStateToProps` which is a good sign that this component got dumber!